plutus-script-utils/src/Plutus/Script/Utils/Typed.hs

```diff
@@ -110,3 +119,129 @@ vForwardingMintingPolicy = tvForwardingMPS
 --   validator
 forwardingMintingPolicyHash :: TypedValidator a -> PV1.MintingPolicyHash
 forwardingMintingPolicyHash = tvForwardingMPSHash
+
+class PV1.UnsafeFromData a => ScriptContext a where
```

Comment:
`IsScriptContext` perhaps? And I wouldn't use `a` here, that's more for type variables that can be (almost) anything.